api/subscribe/subscribe.go

```diff
@@ -0,0 +1,39 @@
+package subscribe
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func PostSubscribe(c echo.Context) error {
+	req := new(SubscribeRequestBody)
```

Comment:
như này nó dễ nhận diện pointer hơn là new, `new` thì nhìn như hàm, `&` thì nhìn biết ngay

```suggestion
	req := &SubscribeRequestBody{}
```